src/gc/gc.d

```diff
@@ -2313,16 +2313,18 @@ struct Gcx
 
         // Convert page to free list
         size_t size = binsize[bin];
-        List **b = &bucket[bin];
+        auto tail = &bucket[bin];
```

Comment:
IIUC this (ab)uses the fact that bucket[bin] is probably null if allocPage is called. I don't think that should be assumed, or at least made clearer by the implementation or a comment.